### ISSUE TITLE
Alt text for icons and images

### ISSUE THREAD

**Issue by bluethreadmade** [bluethreadmade/challenge-1#3]
WHEN I view the icon and image elements
THEN I find accessible alt attributes

**Comment by bluethreadmade** [comment]
images:
- [x] hero
- [x] seo
- [x] reputation
- [x] social media

Icons:
- [x] lead generation
- [x] brand awareness
- [x] cost management

**Comment by bluethreadmade** [comment]
The hero/main image is a background image and I cannot directly add alt text to it. There are workarounds to explore

**Comment by bluethreadmade** [comment]
hero image changed to a figure element to add alt. commit e8548ca
